README.md

```diff
@@ -1 +1,37 @@
-# graal-patches
\ No newline at end of file
+# Graal Patches
+The Graal Patches project builds and publishes the artifact `graal-sdk.tar.gz` required by [Conclave SDK](https://github.com/R3Conclave/conclave-sdk).
+The artifact is based on [Oracle's GraalVM](https://github.com/oracle/graal). 
+
+## Project structure
+The files inside this project are grouped to make the project discovery easy.
+
+### Containers
+The folder 'containers' contains the Dockerfile used to generate the docker container image required to build Graal without
+having to worry about installing any required tools in your system.
+
+### Patches
+The folder 'patches' contains the patch that updates [Oracle's GraalVM](https://github.com/oracle/graal) source code as 
+required by [Conclave SDK](https://github.com/R3Conclave/conclave-sdk).
+
+### Scripts
+The folder 'scripts' groups all the scripts that are required to build the artifact `graal-sdk.tar.gz`. However, you must respect the
```

Comment:
I've noticed that we use this term with "-sdk" for the final artifact (probably at my suggestion). But I think it's wrong. Firstly the Oracle GraalVM distributions do not have "sdk" term, and rather the "sdk" is a specific component of GraalVM, which is the SDK library, as used by [`conclave-enclave`](https://github.com/R3Conclave/conclave-sdk/blob/master/conclave-enclave/build.gradle#L14). Thus we should use the same term as the original which is "graalvm.tar.gz"

Reply:
See https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-22.1.0